build/voltage-swap-default.tokenlist.json

```diff
@@ -246,6 +254,14 @@
       "chainId": 122,
       "logoURI": "https://raw.githubusercontent.com/voltfinance/swap-default-token-list/master/logos/0x3c5DD992D72D082B44c7505015E09fdb1e975eC2/logo.png"
     },
+    {
+      "name": "Affine ultraETHs 2.0",
+      "address": "0xb10eD6E3810c95a380E4f3e448af1755fa3368cf",
+      "symbol": "ultraETHs",
+      "decimals": 18,
+      "chainId": 122,
+      "logoURI": "https://raw.githubusercontent.com/voltfinance/swap-default-token-list/master/logos/0xF0a949B935e367A94cDFe0F2A54892C2BC7b2131/logo.png"
```

Comment:
we have to place the token logo in the folder with the address of the token so the folder needs to be renamed to 0xb10eD6E3810c95a380E4f3e448af1755fa3368cf

please do the same with the other token